orderer/consensus/smartbft/chain.go

```diff
@@ -64,20 +68,82 @@ type signerSerializer interface {
 	Serialize() ([]byte, error)
 }
 
+//go:generate mockery --dir . --name ConsenterSupport --case underscore --with-expecter=true --output mocks
+
+// ConsenterSupport provides the resources available to a Consenter implementation.
+type ConsenterSupport interface {
```

Comment:
this mock already exists here:
`orderer/consensus/mocks/mock_consenter_support.go`
This is a rare case where you don't need to recreate the mock.